### ISSUE TITLE
About performance on ARkitscene validation set

### ISSUE THREAD

**Issue by yifanjiang19** [nianticlabs/simplerecon#20]
Hi,

Thanks for providing the dataloader for ARKitscene dataset. Have you tested on ARKitscene validation set? Expect to see the number!

Best,

**Comment by mohammed-amr** [comment]
That dataloader is for the ARKit format from ios-logger formatted using the scripts in NeuralRecon.

It is not ARKitScenes.
